vpnglean/glean_parser_ext/templates/rust.jinja2

```diff
@@ -26,7 +31,7 @@ Jinja2 template is not. Please file bugs! #}
 #[derive(Default, Debug, Clone, Hash, Eq, PartialEq)]
 pub struct {{ obj.name|Camelize }}{{ suffix }} {
     {% for item, type in obj|attr(name) %}
-    pub {{ item|snake_case }}: Option<{{type|extra_type_name}}>,
+    pub r#{{ item|snake_case }}: Option<{{type|extra_type_name}}>,
```

Comment:
This might be something interesting to incorporate on mozilla-central as well. It addresses the case when for example `type` is one of the event extras. That will fail because `type` is a reserved keyword on Rust. This fixes it.

Reply:
Would you like to file the bug, or would you like me to do it?

Reply:
There you go: https://bugzilla.mozilla.org/show_bug.cgi?id=1800862